src/views/DrefApplicationForm/import.ts

```diff
@@ -1,652 +0,0 @@
-import {
```

Comment:
why is this removed?

Reply:
It had tons of error which was blocking the build